cmd/builder/internal/builder/config.go

```diff
@@ -16,7 +16,7 @@ import (
 	"go.uber.org/zap"
 )
 
-const defaultOtelColVersion = "0.112.0"
+const defaultOtelColVersion = "v0.112.0"
```

Comment:
Does the release automation that bumps this version work well with adding the `v` here?

Reply:
Tested, it works, see where the versions don't have the `v` prefix but it doesn't matter:

https://github.com/open-telemetry/opentelemetry-collector/blob/main/Makefile#L498